plugins/woocommerce-blocks/assets/js/blocks/checkout/address-card/style.scss

```diff
@@ -35,7 +35,7 @@
 	}
 }
 .wc-block-components-address-card__edit {
+	background-color: transparent;
```

Comment:
@nielslange I made the button style consistent with the "+Add apartament..." one: 
![image](https://github.com/user-attachments/assets/dbe6c1a9-50a2-4e86-89bb-020cd937d3e0)

**Hover state**:
![image](https://github.com/user-attachments/assets/9a08b338-b9e2-48fd-a549-874c3972b795)

Reply:
Thanks for reaching out, @Manussakis. IINM, the `+ Add apartment, suite, etc.` button should be styled like a regular link, and only behave like a button in terms of keyboard use. @nikkivias, can you confirm this?

Given that you're working on this section, do you think you could also integrate #49482 in your PR? That issue ensures that the button text remains readable independent of the used theme. I've briefly looked into this issue and I believe the issue could be addressed by adding the following two definitions to `.wc-block-components-address-form__address_2-toggle`:

```css
color: inherit;
font-family: inherit;
```

Reply:
Thanks for the feedback, @nielslange .
I have addressed it, please review take a look at [Adjust checkout toggler buttons for dark themes](https://github.com/woocommerce/woocommerce/pull/49471/commits/738bc0c609a6d3826c38b586f36faced36891e98)

Reply:
Thanks for addressing my concerns, @Manussakis.